distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-frameworks-contents.adoc

```diff
@@ -0,0 +1,9 @@
+
+=== Included Catalog Frameworks
```

Comment:
This title is a little misleading since we aren't really listing included Catalog Frameworks. Could add `and associated components and configurations`?